pages/common/stolonctl.md

```diff
@@ -0,0 +1,19 @@
+# Stolon
+
+> A cloud native PostgreSQL manager for PostgreSQL high availability.
+
+- Get cluster status:
+
+`stolonctl --cluster-name ${{cluster_name}} --store-backend ${{store_backend}} --store-endpoints ${{store_endpoints}} status`
```

Comment:
Are the dollar signs `$` are required when calling the command?

Reply:
@sbrl no they will be needed only if there is a variable substitution. Should I remove the `$` symbol?

Reply:
Yes, please remove it.

Reply:
@agnivade @sbrl I removed it.